File: JS/PGS_3진법 뒤집기.js
function solution(n) {
  var answer = 0;
  let res = "";
  while (n >= 3) {
    l = n % 3;
    n = Math.floor(n / 3);
    res = l + res;
  }
  res = String(n) + res;
  for (let i = 0; i < res.length; i++) {
    answer += Number(res[i]) * 3 ** i;
  }
  return answer;
}

File: Python/PGS_3진법 뒤집기.py
def solution(n):
    answer = 0
    res = ''
    while n >= 3:
        l = n % 3
        n = n // 3
        res = str(l) + res
    res = list(str(n) + res)
    
    for i in range(len(res)):
        answer += int(res[i])*(3**i)
    
    return answer
